plugins/formal-verification/testData/diagnostics/bad_contracts/calls_in_place.fir.diag.txt

```diff
@@ -27,8 +27,7 @@ method global$fun_unknown$fun_take$fun_take$T_Int$return$T_Int$return$T_Int(loca
   ensures true
 
 
-/calls_in_place.kt:(450,472): warning: Viper verification error: Postcondition of global$fun_incorrect_at_most_once$fun_take$fun_take$T_Int$return$T_Int$return$T_Int might not hold. Assertion local$f.special$function_object_call_counter <= old(local$f.special$function_object_call_counter) + 1 might not hold. (<no position>)
-
+/calls_in_place.kt:(450,472): warning: Viper verification error: [postcondition.violated:assertion.false] Postcondition of global$fun_incorrect_at_most_once$fun_take$fun_take$T_Int$return$T_Int$return$T_Int might not hold. Assertion local$f.special$function_object_call_counter <= old(local$f.special$function_object_call_counter) + 1 might not hold. (<no position>)
```

Comment:
I would expect the main result of this change to be that these warnings would now contain a position.  Is the added information useful if Viper can't access it?